plugins/mongodb/mongo_lag

```diff
@@ -25,7 +27,13 @@ import pymongo
 def _get_members():
     host = os.environ.get('host', '127.0.0.1')
     port = os.environ.get('port', 27017)
-    conn = pymongo.MongoClient(host,port)
+    username = os.environ.get('username', '')
+    password = os.environ.get('password', '')
+    conn = pymongo.MongoClient(host,int(port))
```

Comment:
Thank you for your contribution!

Please add a space after the comma (see PEP8). Besides this everything looks fine to me.

Reply:
Should be fine now ?